.NET/Microsoft.Recognizers.Text.NumberWithUnit/NumberWithUnitRecognizer.cs

```diff
@@ -227,6 +227,46 @@ private NumberWithUnitRecognizer(NumberWithUnitOptions options)
                             }
                             ),
             });
+
+            RegisterModel(Culture.German, options.ToString(), new Dictionary<Type, IModel>

```

Comment:
The model registration in the *Recognizer.cs files should only be added once the full entities are supported. Otherwise it will expose detection in a language that might not be ready.

Reply:
Can you remove this for now if there's still any pending units case?
Something like "twenty three euros" in German wouldn't work yest, right?